packages/cli-kit/src/public/node/output.ts

```diff
@@ -245,20 +245,32 @@ export function collectLog(key: string, content: OutputMessage): void {
 }
 
 export const clearCollectedLogs = (): void => {
-  // console.log('clearCollectLogs')
   collectedLogs = {}
-  // console.log(collectedLogs)
 }
 
 /**
- * Ouputs information to the user.
+ * Outputs command result information to the user.
+ * Result messages don't get additional formatting.
+ * Note: By default, result messages are logged at info level to stdout.
+ *
+ * @param content - The content to be output to the user.
+ * @param logger - The logging function to use to output to the user.
+ */
+export function outputResult(content: OutputMessage, logger: Logger = consoleLog): void {
```

Comment:
I want to suggest we avoid having positional params with defaults. It's possible we'd want to add params in the future, and that means passing around a lot of `undefined`s. Especially for such a central function, a bit of extra design is very worthwhile.

As an aside, the param doesn't seem to be used at all. Is the idea to follow up with moving things to `consoleErr` gradually? A default seems to drive against that, instead encouraging continued use of stdout instead of stderr.

Reply:
Yeah, it's unused. I guess Gordon added it to follow the same shape of the other outputX functions, where the logger is passed in some cases. I also agree with using named params, but the refactor is getting too big, I'd prefer to visit it later.

Reply:
If it's unused, can we just remove it for now? Easier to make the interface thin now and then add options later.

Reply:
Done!